cmd/feature.go

```diff
@@ -0,0 +1,194 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/cozy/cozy-stack/client/request"
+	"github.com/spf13/cobra"
+)
+
+var featureCmdGroup = &cobra.Command{
+	Use:   "feature <command>",
+	Short: "Manage the feature flags",
+}
+
+var featureFlagCmd = &cobra.Command{
+	Use:   "flags",
+	Short: `Display and update the feature flags for an instance`,
+	Long: `
+cozy-stack feature flags displays the feature flags that are specific to an instance.
```

Comment:
IIUC this command let you get the instance-specifig flags only. IMHO we should also have a cozy-stack command to display the resulting flags for an instance applying defaults, context and feature-set herited flags. either with a switch to this command or with another command.